### ISSUE TITLE
Pause radio when using wired headphones and disconnecting them

### ISSUE THREAD

**Issue by tanty** [segler-alex/RadioDroid#92]
I am using RadioDroid with headphones. Ideally, there should be a setting to pause the playback when disconnecting the cable of the headphones from the phone.

Also, maybe, another setting to resume the playback if the headphones get plugged back.

Thanks!

**Comment by davidhedlund** [comment]
Yeah, its simply not convenient to listen to music without this feature. If the headphones cable loosening it will be embarrassing if use on boring events where the audience is silent. Everyone will look at you.

**Comment by tanty** [comment]
#22 is similar, maybe this can be closed as DUPLICATED.

**Comment by davidhedlund** [comment]
@tanty Thank you!

**Comment by mxmilkb** [comment]
A feature/option to resume on reconnecting headphones would be very handy.
